ThirdPartyModules/Magento/InStorePickupShipping.php

```diff
@@ -321,6 +321,7 @@ public function setInStoreShippingAddressForPrepareQuote(
                     SalesChannelInterface::TYPE_WEBSITE,
                     $quote->getStore()->getWebsite()->getCode()
                 );
+                $extraData = ['telephone'=>$pickupLocation->getPhone()];
```

Comment:
Does not seem used anywhere.

Reply:
@danac-gs OH, my bad, something is missing when copying the files from my dev server to local. Fixed